src/index.css

```diff
@@ -56,6 +57,10 @@ blockquote {
 	height: 14px;
 }
 
+.components-tab-panel__tabs--assistant:hover:not(:active)::before {
```

Comment:
Drive by fix to ensure that the icon in the assistant tab goes blue on hover.

<img width="143" alt="Screenshot 2024-06-25 at 11 39 55" src="https://github.com/Automattic/studio/assets/417538/2898ff31-70c9-4669-97b0-80bc1eaae969">